src/Cache.js

```diff
@@ -1,55 +1,38 @@
 'use strict'
 
-const fs   = require('fs')
-const path = require('path')
-
 let filePath
 let cache = {}
 
-class Cache {
-  static set(key, value) {
-    return new Promise((resolve, reject) => {
-      cache[key] = value
-      if(filePath) {
-        // console.log("write cache:", filePath, JSON.stringify(cache, null, 2))
-        fs.writeFile(filePath, JSON.stringify(cache, null, 2) + "\n", resolve)
-      } else {
-        resolve()
-      }
-    })
-  }
+let store = typeof window !== 'undefined' && window.localStorage
+
+if (!store) {
+  const LocalStorage = require('node-localstorage').LocalStorage
+  store = new LocalStorage('./scratch')
+}
```

Comment:
We should be able to define the path where the data is saved. 

Perhaps a solution here would be to create the instance in loadCache() and use filePath - (filename.ext) as the data directory, and filename.ext as the key?